tendrl/integrations/gluster/alerts/__init__.py

```diff
@@ -4,23 +4,28 @@
 def update_alert_count(alert):
     if alert.resource in NS.integrations.gluster.objects.VolumeAlertCounters(
             )._defs['relationship'][alert.alert_type.lower()]:
-        alert.tags['volume_id'] = find_volume_id(
+        volume = find_volume_id(
```

Comment:
function can rather be called `find_volume_with_name` I feel

Reply:
ok, i will change this